Barycentric_coordinates_2/test/Barycentric_coordinates_2/test_dh_deprecated_api.cpp

```diff
@@ -0,0 +1,74 @@
+#include <cmath>
+#include <cassert>
+#include <CGAL/Exact_predicates_exact_constructions_kernel.h>
```

Comment:
```suggestion
#include <CGAL/internal/disable_deprecation_warnings_and_errors.h>
#include <CGAL/Exact_predicates_exact_constructions_kernel.h>
```

All tests of deprecated functions or classes should include that header `<CGAL/internal/disable_deprecation_warnings_and_errors.h>`, to remove warnings or errors about deprecated code.

Reply:
Oo, I did not know that. Thanks. I will fix that.

Reply:
Fixed.